pages/studentPortal/quizcomponent/tech-per-quiz.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import { Button } from "../../../components/ui/Button";
+
+const Intake = () => {
+    const [onBoarding, setOnBoarding] = useState({
+        name: "",
+        email: ""
+    });
+
+    const handleChange = (e) => {
+        const { name, value } = e.target
+        setOnBoarding((prev) => {
+            return { ...prev, [name]: value };
+        });
+    };
+
+    const handleSubmit = (e) => {
+        // e.preventDefault();
```

Comment:
it's bad practice to commit commented code. better to remove this function entirely for now, if it's not needed at the moment. you can create a follow up pr to wire up the submit button.